components/solr-access-service/ui/src/main/resources/PhenoTips/SolrService.xml

```diff
@@ -41,15 +41,23 @@
 #if ($xcontext.action == 'get')
   #set ($results = [])
   #if ("$!{request.q}" != '')
+    #set ($filter = "$!{request.fq}")
     #set ($vocabulary = "$!{request.vocabulary}")
     #if ($vocabulary == '')
       #set ($vocabulary = 'hpo')
     #end
+    #if ($filter == '')
+      #if ($vocabulary == 'hpo')
+        #set ($filter = 'term_category:HP\:0000118')
+      #elseif ($vocabulary == 'omim')
+        #set ($filter = '-(nameSort:\** nameSort:\+* nameSort:\^*)')
```

Comment:
This isn't good:
- it's code duplication
- it's an if-elseif anti-pattern that will grow with each new vocabulary

The refactoring isn't needed this, but changing the places where this page is used to instead specify a category instead of a vocabulary.